src/components/buttons/save-button/save-button.js

```diff
@@ -1,29 +1,72 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { Button } from '@material-ui/core';
 import PropTypes from 'prop-types';
+import { config } from '../../../configs';
 
-const SaveButton = ({ title, type, onClickHandler, color, ...props }) => (
-  <Button
-    variant='contained'
-    color={color}
-    type={type}
-    onClick={onClickHandler}
-    {...props}
-  >
-    {title}
-  </Button>
-);
+const SaveButton = ({
+  title,
+  type,
+  onClickHandler,
+  color,
+  errors,
+  values,
+  ...props
+}) => {
+  const { IMG_URL } = config;
+  const error = !!Object.keys(errors).length;
+  const disable = Object.values(values).every((el) => {
+    console.log(el);
+    if (typeof el === 'boolean') {
+      return true;
+    }
+    if (el) {
+      if (el === IMG_URL) {
```

Comment:
We don't need this IMG_URL in categories form

Reply:
fixed